src/app/services/tests/AuthenticationService.spec.ts

```diff
@@ -12,9 +12,12 @@ import { setupEmulators } from 'src/app/utils/tests/TestUtils.spec';
 
 class RTDBSpec {
     // TODO: these are stubs that can be removed after the RTDB functions ticket has been done
-    public static clearDB(): Promise<void> {
-        // Here we can't clear the DB because it breaks everything, but this is how it should be done:
-        //  await firebase.database().ref().set(null);
+    public static async clearDB(): Promise<void> {
+        const useFirebaseDatabase: boolean = false;
+        if (useFirebaseDatabase) {
```

Comment:
eh ok, so if the point is not to use that function, good to know but ... shouldn't we delete it ?

Reply:
(the function)

Reply:
non, elle est mockée

Reply:
fait !